1-exercises/D-array-filter/exercise.js

```diff
@@ -8,7 +8,10 @@
 
 let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw.filter((item) => !!item && item.length === 2); 
```

Comment:
`!!` is nice shorthand for checking for a variables existence, however be wary this the `!` operator will actually check for [Falsey](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values when used this way. This means that, if `0` is a valid value, `!0` evaluates to `true` as `0` is a falsey value.